test/functional/apps/discover/_context_encoded_url_param.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const dataGrid = getService('dataGrid');
+  const kibanaServer = getService('kibanaServer');
+  const PageObjects = getPageObjects(['common', 'discover', 'timePicker', 'settings', 'header']);
+  const testSubjects = getService('testSubjects');
+  const es = getService('es');
+
+  describe('context encoded id param', () => {
+    before(async function () {
+      await PageObjects.common.navigateToApp('settings');
+      await es.transport.request({
+        path: '/includes-plus-symbol-doc-id/_doc/1+1=2',
+        method: 'PUT',
+        body: {
+          username: 'matt',
+          '@timestamp': '2015-09-21T09:30:23',
+        },
+      });
+      await PageObjects.settings.createIndexPattern('includes-plus-symbol-doc-id');
```

Comment:
didn't have this way to create testdata on the radar, TIL. only disadvantage is, that it might be slower, more UI interaction needed to create the required testing data but, fine with that. 
only nit: username should be `Dmitry`, the `matt` you are mentioning here just added comments here, else he was lazy as usual 😄.